### ISSUE TITLE
Allow nodes to display their values

### ISSUE THREAD

**Issue by firstdorsal** [chrisjpatty/flume#169]
I would love to see the intermediate values of nodes. 
Currently one has to backtrace the nodes to see the original input from one node.

**Comment by osafafi** [comment]
@firstdorsal and have you found a workaround this?
I'm interested in having the values of each port displayed as overlay on top of it. that would be very helpful as a debug tool.
Any idea? @chrisjpatty

**Comment by firstdorsal** [comment]
I don't think this is possible without introducing major changes.

**Comment by osafafi** [comment]
yeah i get that. You think it's too complicated or not worth it to expose the dom elements for each node? cause then we can do all sorts of things with (append to for example)
